packages/packages/locations/src/components/slot.tsx

```diff
@@ -1,20 +1,14 @@
-import { Suspense } from 'react';
-import useFills from '../hooks/use-fills';
+import * as React from 'react';
+import useInjectionsOf from '../hooks/use-injections-of';
+import { Location } from '../types';
 
-type SlotProps = {
-	location: string;
-}
-
-// TODO: <ErrorBoundary />
-export default function Slot( { location }: SlotProps ) {
-	const fillers = useFills( location );
+export default function Slot( { location }: { location: Location } ) {
```

Comment:
maybe extract to SlotProps for reabaility?

Reply:
Done